dev/com.ibm.ws.org.apache.cxf.ws.security.2.6.2/bnd.bnd

```diff
@@ -0,0 +1,39 @@
+#*******************************************************************************
+# Copyright (c) 2020 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+#*******************************************************************************
+-include= jar:${fileuri;${repo;org.apache.cxf:cxf-rt-ws-security;2.6.2;EXACT}}!/META-INF/MANIFEST.MF,bnd.overrides
+
+globalize: false
+instrument.ffdc: true
+
+cxfVersion=2.6.2
+
+-includeresource: \
+    @${repo;org.apache.cxf:cxf-rt-ws-security;${cxfVersion};EXACT}!/!META-INF/maven/*|\
```

Comment:
@${repo;org.apache.cxf:cxf-rt-ws-security;${cxfVersion};EXACT}!/!META-INF/maven/*|\ is the | at the end of the line valid? There is no comma either. If the | character is valid then it should be maven/*| , \

Reply:
This was same at com.ibm.ws.org.apache.cxf.ws.security/bnd.bnd. Should I change?